mobile/src/screens/ConsumerRegistration.tsx

```diff
@@ -153,7 +164,12 @@ function ConsumerRegistration() {
                 <Input
                   value={lastName}
                   accessibilityLabel="last name"
-                  onChangeText={setLastName}
+                  aria-label="Last name"
+                  onChangeText={(text) => {
+                    if (isValidNameInput(text) || text === "") {
```

Comment:
I see `isValidNameInput(text) || text === ""` is everytime being used in combination. Could you rather then update the `isValidNameInput(text)` method to first check if the `text` is `""`?

Reply:
Fixed.

Reply:
Thanks